src/ralph/licences/migrations/0006_auto_20200908_0847.py

```diff
@@ -0,0 +1,26 @@
+        ),
+        migrations.AlterField(
```

Comment:
Will this convert existing values with no data loss?

Reply:
Yes. `MoneyField` extends `DecimalField`. Precision and default value stay the same; field name (`price`) also remains the same. In the database, currency is stored in a separate field (`price_currency`). Also, in the REST API field `price` looks the same as before; currency is displayed in a separate `price_currency` fideld.